sdks/java/core/src/main/java/org/apache/beam/sdk/util/MergingActiveWindowSet.java

```diff
@@ -144,7 +144,7 @@ public void ensureWindowIsActive(W window) {
     checkState(stateAddressWindows != null,
                              "Cannot ensure window %s is active since it is neither ACTIVE nor NEW",
                              window);
-    if (stateAddressWindows.isEmpty()) {
+    if (stateAddressWindows != null && stateAddressWindows.isEmpty()) {
```

Comment:
This is redundant due to the check above.

I don't see Eclipse complaining about this one in my client. What is the error?